src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -2172,16 +2311,64 @@
           short-summary: Name or ID of the VM to target.
         - name: --filters
           short-summary: JSON encoded list of packet filters. Use `@<file path>` to load from file.
+    examples:
+        - name: Create a packet capture session on a VM
+          text: az network watcher packet-capture create -g {resoureceurceGroupName} -n packetCaptureName --vm {vmName} --storage-account gwteststorage123abc
+        - name: Create a packet capture session on a VM with optional filters
+          text: az network watcher packet-capture create -g {resoureceurceGroupName} -n packetCaptureName --vm {vmName} --storage-account gwteststorage123abc --filters "[{\"protocol\":\"TCP\", \"remoteIPAddress\":\"1.1.1.1-255.255.255\",\"localIPAddress\":\"10.0.0.3\", \"remotePort\":\"20\"},{\"protocol\":\"TCP\", \"remoteIPAddress\":\"1.1.1.1-255.255.255\",\"localIPAddress\":\"10.0.0.3\", \"remotePort\":\"80\"},{\"protocol\":\"TCP\", \"remoteIPAddress\":\"1.1.1.1-255.255.255\",\"localIPAddress\":\"10.0.0.3\", \"remotePort\":\"443\"},{\"protocol\":\"UDP\"}]"
+"""
+
+helps['network watcher packet-capture delete'] = """
+    type: command
+    long-summary: For more information about configuring packet capture visit https://docs.microsoft.com/en-us/azure/network-watcher/network-watcher-packet-capture-manage-cli
+    examples:
+        - name: Delete a packet capture session. Note that this only deletes the session and not the capture file.
```

Comment:
Remove `Note that`

Reply:
This style of comment is more appropriate for a `long-summary`. If sessions are deleted, but not capture files, that should be information contained in the help itself. This also shows that the example could be improved, by deleting the capture file (if easily demonstrated.)

Reply:
Addressed.